### ISSUE TITLE
unknown block type deployment_circuit_breaker

### ISSUE THREAD

**Issue by si14** [VladRassokhin/intellij-hcl#346]
### Prerequisites
 - [x] Ensure you have latest version of plugin installed
 - [x] Search for possible issue duplicates

### Installation details
 - [x] IDE version (`Help->About->Copy to Clipboard`)
 - [x] intellij-hcl plugin version (`Settings->Plugins`)
 - [x] Terraform version (`terraform -v`)

IDE version: 

```
PyCharm 2021.1.1 (Professional Edition)
Build #PY-211.7142.13, built on April 21, 2021
Licensed to …
You have a perpetual fallback license for this version.
Subscription is active until May 22, 2022.
Runtime version: 11.0.10+9-b1341.41 x86_64
VM: Dynamic Code Evolution 64-Bit Server VM by JetBrains s.r.o.
macOS 11.4
GC: G1 Young Generation, G1 Old Generation
Memory: 1126M
Cores: 16
Non-Bundled Plugins: org.intellij.plugins.hcl (0.7.10), mobi.hsz.idea.gitignore (4.1.0), name.kropp.intellij.makefile (211.6693.108), org.logtalk (1.0.15), org.toml.lang (0.2.150.3968-211), rocks.blackcat.vcl (0.7.2), idea.plugin.protoeditor (2.3.1)
```

Plugin version: 0.7.10

Terraform version: 1.0.2

### Terraform Configuration Files
```hcl
resource "aws_ecs_service" "test_service" {
  name = "test-name"
  cluster = "foo"
  task_definition = "bar:1"
  desired_count = 1
  
  deployment_circuit_breaker {
    enable = true
    rollback = true
  }
}
```

### Expected Behavior
`deployment_circuit_breaker` should be recognised (see https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_service#deployment_circuit_breaker for the definition)

### Actual Behavior
`deployment_circuit_breaker` gets highlighted as an error with `unknown block type deployment_circuit_breaker`

**Comment by si14** [comment]
Update: I solved this by following https://github.com/VladRassokhin/terraform-metadata/blob/master/schemas-extractor/ReamMe.md , which probably means that that repository needs to be updated.

**Comment by VladRassokhin** [comment]
Fixed in 0.7.11
